engine.go

```diff
@@ -53,21 +55,51 @@ type OpenAPIConfig struct {
 	DisableLocalSave bool
 	// Pretty prints the OpenAPI spec with proper JSON indentation
 	PrettyFormatJSON bool
+	// URL to serve the OpenAPI JSON spec
+	SpecURL string
+	// Handler to serve the OpenAPI UI from spec URL
+	UIHandler func(specURL string) func(c ContextNoBody) (HTML, error)
```

Comment:
Changed this to adhere to the fuego controller func. Think it's a nice way to introduce someone to a fuego native route. It also allows us to unify this across adaptors as every adaptor will need a `handlefuego`

Reply:
We actually can't do this as it makes it impossible (well maybe not impossible but a lot more code in other places to save a few lines in others) to add middleware specific just to this handler.

Reply:
Oooh too bad :c

It was a good idea indeed